html/syntax/parsing/template/additions-to-the-in-frameset-insertion-mode/end-tag-frameset.html

```diff
@@ -18,15 +18,9 @@
     var doc = context.iframes[0].contentDocument;
 
     var frameset = doc.querySelector('frameset');
-    assert_equals(frameset.children.length, 1, 'Wrong number of frameset children elements');
+    assert_equals(frameset.children.length, 0, 'Wrong number of frameset children elements');
 
-    var template = frameset.querySelector('template');
-    assert_equals(template.tagName, 'TEMPLATE', 'FRAMESET should contain template element');
-    assert_equals(template.content.childNodes.length, 0,
-            'Template content should be empty');
-
-}, '</frameset> tag should be ignored if there\'s TEMPLATE element in '
-    + 'the stack of open elements');
+}, '<template> tag should be ignored if there\'s TEMPLATE element in the stack of open elements');
```

Comment:
The test name looks wrong. `'<template> tag should be ignored in "in frameset" insertion mode'`?